Makefile

```diff
@@ -93,6 +100,13 @@ configure-installer-tests-cluster:
 test:
 	go test -race $(PKGS)
 
+# This test is only for reference and will be removed
+# in the subsequent subtask of https://github.com/openshift/odo/issues/1473 
+# Run clean test template spec parallelly
+.PHONY: clean-test
+clean-test:
+	ginkgo -v -nodes=$(TEST_EXEC_NODES) -focus="Example of a clean test" slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -randomizeAllSpecs  tests/template/ -timeout $(TIMEOUT)
```

Comment:
I'm still not a big fan of this. I don't see why we need it here if it will never bee executed and it is used just as documentation on how tests should be run?

I think that a better place for this is in the test template, or in docs.

Reply:
Done